src/components/item.jsx

```diff
@@ -0,0 +1,27 @@
+import bag_1 from "../assets/bag_1.png"
+import bag_2 from "../assets/bag_2.png"
+import bag_3 from "../assets/bag_3.png"
+import bag_4 from "../assets/bag_4.png"
+
+export default function Item(props) {
+    let image;
```

Comment:
Je hoeft hier geen variabele en if-else statement te gebruiken. Je kunt de afbeeldingen gewoon in App.jsx importeren en daar aan de image property meegeven. 

Dan krijg je in App.jsx:

```suggestion
import bag_1 from "../assets/bag_1.png"
     
     
<Item label="New collection"
           img={bag_4}
           name="The trendy bag"
           price="€150,-" />
```